ui/src/VoxettaRecordButton.js

```diff
@@ -13,39 +13,56 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
-import {AudioRecorder} from './AudioRecorder';
 import {LitElement, html, css} from 'lit-element';
 
+import {AudioRecorder} from './AudioRecorder';
+import {UtteranceApiService} from './UtteranceApiService';
+
+/**
+ * Button responsible for enabling the user to record and upload audio files. 
+ */
 export class VoxettaRecordButton extends LitElement {
     static get properties() {
         return {
             isRecording: {type: Boolean},
             audioRecorder: {type: Object},
+            utteranceService: {type: Object}
```

Comment:
audioRecorder and utteranceService don't need to be LitElement properties. Only things that should trigger a component redraw should be included in this map. These can be just plain javascript properties (can attach to `this` but don't need to list in `static get properties`)

I assume isRecording would eventually change the button color or something like that so it's OK to stay a LitElement property.

Reply:
This makes sense--removed